mllib/src/test/scala/org/apache/spark/ml/regression/DecisionTreeRegressorSuite.scala

```diff
@@ -96,6 +108,15 @@ class DecisionTreeRegressorSuite
       assert(variance === expectedVariance,
         s"Expected variance $expectedVariance but got $variance.")
     }
+
+    val toyDF = TreeTests.setMetadata(toyData, Map.empty[Int, Int], 0)
+    dt.setMaxDepth(1)
+      .setMaxBins(6)
```

Comment:
Ok, not sure it needs to be set since the test is fine without it. The warning is more just helpful info logging. I'm fine to leave it since it doesn't matter either way.